outscale/resource_outscale_load_balancer.go

```diff
@@ -447,6 +452,53 @@ func resourceOutscaleOAPILoadBalancerCreate_(d *schema.ResourceData, meta interf
 		return err
 	}
 
+	scVal, scOk := d.GetOkExists("secured_cookies")
+
+	if err := resourceOutscaleOAPILoadBalancerRead(d, meta); err != nil {
+		return nil
+	}
+
+	d.Partial(true)
+	d.SetPartial("subregion_names")
+	d.SetPartial("load_balancer_name")
+	d.SetPartial("load_balancer_type")
+	d.SetPartial("security_groups")
+	d.SetPartial("subnets")
+	d.SetPartial("tags")
+	d.SetPartial("dns_name")
+	d.SetPartial("access_log")
+	d.SetPartial("health_check")
+	d.SetPartial("backend_vm_ids")
+	d.SetPartial("listeners")
+	d.SetPartial("source_security_group")
+	d.SetPartial("public_ip")
+	d.SetPartial("net_id")
+	d.SetPartial("application_sticky_cookie_policies")
+	d.SetPartial("load_balancer_sticky_cookie_policies")
+
```

Comment:
``` d.SetPartial ``` is deprecated in the next version of terraform-SDK, if we can avoid to use it now that will be cool

Reply:
Ok I haven't see that, i remove this ^^